spring-integration-hazelcast/src/main/java/org/springframework/integration/hazelcast/leader/LeaderInitiator.java

```diff
@@ -131,7 +130,7 @@ public LeaderInitiator(HazelcastInstance client, Candidate candidate) {
 		this.client = client;
 		this.candidate = candidate;
 	}
-
+	
```

Comment:
Please, make sure there are no trailing whitespaces.
You can run `./gradlew check` to be sure that your changes satisfy our Checkstyle rules.